pydy/viz/scene.py

```diff
@@ -462,8 +605,8 @@ def remove_static_html(self, force=False):
 
     def display(self):
         """Displays the scene in the default webbrowser."""
-        self.create_static_html()
-        server = Server(scene_file=self._scene_json_file)
+        self._generate_json()
```

Comment:
Don't you have to supply the directory kwarg here, other wise the json files will not be copied into the `_static_url` directory.